source/includes/iam-user-policy-minimum.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Version": "2012-10-17",
```

Comment:
I believe there [are only 2 valid versions](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html) for the version variable in an IAM JSON policy (`2012-10-17` or `2008-10-17`). As AWS supplies this version for (AFAICT) all of their [sample inline policies](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_examples.html) (even policies that do not make use of [policy variables](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_iam_policy-sim.html)), I believe it would be idiomatic for us to supply this value as well.

Reply:
I see, I was confused by the policy version which is not the same as this (Version policy element).